ui/lib/pki/addon/components/page/pki-issuer-list.hbs

```diff
@@ -65,30 +65,40 @@
               <span class="has-text-weight-semibold is-underline">
                 {{pkiIssuer.issuerRef}}
                 {{#if pkiIssuer.issuerName}}
-                  <span class="tag has-text-grey-dark">{{pkiIssuer.id}}</span>
+                  <Hds::Badge @text={{pkiIssuer.id}} class="has-text-grey-dark" />
                 {{/if}}
               </span>
               <div class="is-flex-row has-left-margin-l has-top-margin-xs">
                 {{#if pkiIssuer.isDefault}}
-                  <span class="tag has-text-grey-dark" data-test-is-default={{idx}}>default issuer</span>
+                  <Hds::Badge
+                    @text="default issuer"
+                    data-test-is-default={{idx}}
+                    class="has-text-grey-dark has-right-margin-xs"
+                  />
                 {{/if}}
                 {{#if (not-eq pkiIssuer.isRoot undefined)}}
-                  <span class="tag has-text-grey-dark" data-test-is-root-tag={{idx}}>{{if
-                      pkiIssuer.isRoot
-                      "root"
-                      "intermediate"
-                    }}</span>
+                  <Hds::Badge
+                    @text={{if pkiIssuer.isRoot "root" "intermediate"}}
+                    data-test-is-root-tag={{idx}}
+                    class="has-text-grey-dark has-right-margin-xs"
+                  />
                 {{/if}}
                 {{#if pkiIssuer.serialNumber}}
-                  <span class="tag is-transparent has-right-margin-none" data-test-serial-number={{idx}}>
+                  <span
+                    class="is-transparent is-flex-v-centered has-right-margin-xs has-side-padding-xs"
```

Comment:
do we need `is-transparent` here? 

I also think we can remove the addition right side margin. I think the space was so big previously because it was the default `tag` spacing, which we don't need to preserve here.